src/Parallax.js

```diff
@@ -1,39 +1,33 @@
-import React, { Component } from 'react';
+import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 
-class Parallax extends Component {
-  componentDidMount() {
-    if (typeof M !== 'undefined') {
-      const { options } = this.props;
-      this.instance = M.Parallax.init(this._parallax, options);
+const Parallax = ({ children, className, image, options, ...props }) => {
+  const _parallax = useRef(null);
+
+  useEffect(() => {
+    let instance = null;
+
+    if (_parallax.current) {
+      if (typeof M !== 'undefined') {
+        instance = M.Parallax.init(_parallax.current, options);
+      }
```

Comment:
I know this was how it was before, but what do you think about removing this check and if for some reason `M` does not exist, it will show an error instead of silently failing?

Reply:
Okay, so you want to delete that check, right?

Reply:
yeah :)

Reply:
Here we go :smile: